### PR TITLE
Microdata changes for main page, blog, pricing

### PR DESCRIPTION
**Pull request by andrewjcasal**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/_includes/head.html [hunk 1]
```diff
@@ -29,6 +29,15 @@
...
   <meta name="twitter:url" content="{{site.url}}{{page.url}}">
 
+  <!-- Microdata -->
+  <link href="https://www.facebook.com/koding" itemprop="sameAs" id="sitelink-profile-facebook">
+  <link href="https://twitter.com/koding" itemprop="sameAs" id="sitelink-profile-twitter">
+  <link href="https://www.linkedin.com/koding" itemprop="sameAs" id="sitelink-profile-linkedin">
+  <link href="https://github.com/koding" itemprop="sameAs" id="sitelink-profile-github">
+  <meta itemscope="" itemtype="http://schema.org/WebSite" itemref="sitename sitelink  sitelink-profile-facebook sitelink-profile-twitter sitelink-profile-linkedin sitelink-profile-github ">
+  <meta itemprop="name" content="Koding" id="sitename">
+  <link href="https://www.koding.com/" itemprop="url" id="sitelink">
```
Comment (sinan) [inline docs/_includes/head.html]: please use `site` variables e.g. `{{ site.url }}`, `{{ site.social.facebook }}` etc.

File: docs/_layouts/post.html [hunk 2]
```diff
@@ -25,20 +25,20 @@
...
 
     <main class="markdown-body">
 
       <!-- POST TITLE -->
-      <h1>{{ page.title }}</h1>
+      <h1 itemprop="headline">{{ page.title }}</h1>
 
       {% if page.image %}
       <!-- POST IMAGE -->
       <figure><img src="{{ site.url }}{{ page.image }}"></figure>
```
Comment (sinan) [inline docs/_layouts/post.html]: this should also have a `itemprop`

File: docs/pricing/index.html [hunk 3]
```diff
@@ -35,182 +35,191 @@ <h1 class="c-heading-1">Pricing</h1>
...
-    <aside class="Pricing-PriceSegments--priceSection">
-      <form>
-        <div class="Dropdown">
-          <a href="#" class="Dropdown-selection">51-100 Monthly Active Developers</a>
-          <div class="Dropdown-options">
-            <a href="#" attr-value="1-10">1-10 Monthly Active Developers</a>
-            <a href="#" attr-value="11-50">11-50 Monthly Active Developers</a>
-            <a href="#" attr-value="51-100">51-100 Monthly Active Developers</a>
-            <a href="#" class="disabled">for 100+ Please contact us...</a>
+  <div itemscope itemtype="http://schema.org/Product">
```
Comment (sinan) [inline docs/pricing/index.html]: you could have done this on wrapper div no?

**Review by sinan — CHANGES_REQUESTED** [review]
it's a good start 👍

File: docs/pricing/index.html (hunk 3 above)
Comment (andrewjcasal) [inline docs/pricing/index.html]: I was thinking about it, but decided that the pricing page captures 4 different "products" (i.e. Dev Environment Automation)

Comment (sinan) [inline docs/pricing/index.html]: you can still do it with the existing markup to those:
img
such as:
```html
 <section id="Pricing-PriceSegments--devTeams" itemscope itemtype="http://schema.org/Product">
```

Comment (andrewjcasal) [inline docs/pricing/index.html]: Ah, makes sense. Will do.

**Review by sinan — APPROVED** [review]
much cleaner thanks!

**Pull request merged**